### PR TITLE
Adding rel notes for v1.2 release. Now with links to the PRs

### PR DESCRIPTION
**Pull request by aburdenthehand**
**What this PR does / why we need it**:

Adding the release notes for v1.2
This version we've also added links to the PRs to make it easier for readers to see more context.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -5,6 +5,100 @@ hide:
...
+
+### SIG-compute
+- [[PR #11054]](https://github.com/kubevirt/kubevirt/pull/11054) [jean-edouard] New cluster-wide `vmRolloutStrategy` setting to define whether changes to VMs should either be always staged or live-updated when possible.
+- [[PR #10961]](https://github.com/kubevirt/kubevirt/pull/10961) [jcanocan] Reduced VM rescheduling time on node failure
+- [[PR #10840]](https://github.com/kubevirt/kubevirt/pull/10840) [acardace] Requests/Limits can now be configured when using CPU/Memory hotplug
+- [[PR #10839]](https://github.com/kubevirt/kubevirt/pull/10839) [RamLavi] Change second emulator thread assign strategy to best-effort.
+- [[PR #10783]](https://github.com/kubevirt/kubevirt/pull/10783) [RamLavi] Support multiple CPUs in Housekeeping cgroup
+
+### SIG-storage
+- [[PR #10598]](https://github.com/kubevirt/kubevirt/pull/10598) [alicefr] Add PVC option to the hook sidecars for supplying additional debugging tools
```
Comment (alicefr) [inline docs/release_notes.md:48]: This is related to sidecars not to storage

File: docs/release_notes.md [hunk 2]
```diff
@@ -5,6 +5,100 @@ hide:
...
+- [[PR #10916]](https://github.com/kubevirt/kubevirt/pull/10916) [orelmisan] Fix the value of VMI `Status.GuestOSInfo.Version`
+- [[PR #10914]](https://github.com/kubevirt/kubevirt/pull/10914) [brianmcarey] KubeVirt is now built with go 1.21.5
+- [[PR #10898]](https://github.com/kubevirt/kubevirt/pull/10898) [matthewei] vmi status's guestOsInfo adds `Machine`
+- [[PR #10879]](https://github.com/kubevirt/kubevirt/pull/10879) [brianmcarey] Built with golang 1.20.12
+- [[PR #10863]](https://github.com/kubevirt/kubevirt/pull/10863) [dhiller] Remove year from generated code copyright
+- [[PR #10809]](https://github.com/kubevirt/kubevirt/pull/10809) [orelmisan] Source virt-launcher: Log migration info by default
+- [[PR #10787]](https://github.com/kubevirt/kubevirt/pull/10787) [matthewei] virtctl support to add template label and annotation filters
+- [[PR #10720]](https://github.com/kubevirt/kubevirt/pull/10720) [awels] Restored hotplug attachment pod request/limit to original value
+- [[PR #10657]](https://github.com/kubevirt/kubevirt/pull/10657) [germag] Exposing Filesystem Persistent Volumes (PVs)  to the VM using unprivilege virtiofsd.
```
Comment (alicefr) [inline docs/release_notes.md:91]: I would put this under storage

File: docs/release_notes.md [hunk 3]
```diff
@@ -5,6 +5,100 @@ hide:
...
+- [[PR #11001]](https://github.com/kubevirt/kubevirt/pull/11001) [fossedihelm] Allow `kubevirt.io:default` clusterRole to get,list kubevirts
+- [[PR #10918]](https://github.com/kubevirt/kubevirt/pull/10918) [orelmisan] VMClone: Emit an event in case restore creation fails
+- [[PR #10916]](https://github.com/kubevirt/kubevirt/pull/10916) [orelmisan] Fix the value of VMI `Status.GuestOSInfo.Version`
+- [[PR #10914]](https://github.com/kubevirt/kubevirt/pull/10914) [brianmcarey] KubeVirt is now built with go 1.21.5
+- [[PR #10898]](https://github.com/kubevirt/kubevirt/pull/10898) [matthewei] vmi status's guestOsInfo adds `Machine`
+- [[PR #10879]](https://github.com/kubevirt/kubevirt/pull/10879) [brianmcarey] Built with golang 1.20.12
+- [[PR #10863]](https://github.com/kubevirt/kubevirt/pull/10863) [dhiller] Remove year from generated code copyright
+- [[PR #10809]](https://github.com/kubevirt/kubevirt/pull/10809) [orelmisan] Source virt-launcher: Log migration info by default
+- [[PR #10787]](https://github.com/kubevirt/kubevirt/pull/10787) [matthewei] virtctl support to add template label and annotation filters
```
Comment (alicefr) [inline docs/release_notes.md:89]: If we have a client sig for virctl then this can go there

File: docs/release_notes.md [hunk 4]
```diff
@@ -5,6 +5,100 @@ hide:
...
+- [[PR #11047]](https://github.com/kubevirt/kubevirt/pull/11047) [jschintag] Fix potential crash when trying to list USB devices on host without any
+- [[PR #11001]](https://github.com/kubevirt/kubevirt/pull/11001) [fossedihelm] Allow `kubevirt.io:default` clusterRole to get,list kubevirts
+- [[PR #10918]](https://github.com/kubevirt/kubevirt/pull/10918) [orelmisan] VMClone: Emit an event in case restore creation fails
+- [[PR #10916]](https://github.com/kubevirt/kubevirt/pull/10916) [orelmisan] Fix the value of VMI `Status.GuestOSInfo.Version`
+- [[PR #10914]](https://github.com/kubevirt/kubevirt/pull/10914) [brianmcarey] KubeVirt is now built with go 1.21.5
+- [[PR #10898]](https://github.com/kubevirt/kubevirt/pull/10898) [matthewei] vmi status's guestOsInfo adds `Machine`
+- [[PR #10879]](https://github.com/kubevirt/kubevirt/pull/10879) [brianmcarey] Built with golang 1.20.12
+- [[PR #10863]](https://github.com/kubevirt/kubevirt/pull/10863) [dhiller] Remove year from generated code copyright
+- [[PR #10809]](https://github.com/kubevirt/kubevirt/pull/10809) [orelmisan] Source virt-launcher: Log migration info by default
```
Comment (alicefr) [inline docs/release_notes.md:88]: This can go in sig-compute

File: docs/release_notes.md (hunk 3 above)
Comment (aburdenthehand) [reply docs/release_notes.md:91]: We don't so I'll leave this here for now. 
I've addressed all other comments - thank you for the review!

**Comment by aburdenthehand** [comment]
@dhiller @AlonaKaplan Can you please take a look and give it your blessing? The 1.2 release is out. Thank you!

File: docs/release_notes.md [hunk 5]
```diff
@@ -5,6 +5,102 @@ hide:
...
+- [[PR #10651]](https://github.com/kubevirt/kubevirt/pull/10651) [machadovilaca] Refactor monitoring  recording-rules
+- [[PR #10570]](https://github.com/kubevirt/kubevirt/pull/10570) [machadovilaca] Fix LowKVMNodesCount not firing
+- [[PR #10418]](https://github.com/kubevirt/kubevirt/pull/10418) [machadovilaca] Add total VMs created metric
+
+### Uncategorized
+- [[PR #11144]](https://github.com/kubevirt/kubevirt/pull/11144) [0xFelix] virtctl: Specifying size when creating a VM and using --volume-import to clone a PVC or a VolumeSnapshot is optional now
+- [[PR #11122]](https://github.com/kubevirt/kubevirt/pull/11122) [brianmcarey] Update runc dependency to v1.1.12
+- [[PR #11068]](https://github.com/kubevirt/kubevirt/pull/11068) [brianmcarey] Update container base image to use current stable debian 12 base
+- [[PR #11050]](https://github.com/kubevirt/kubevirt/pull/11050) [fossedihelm] restrict default cluster role to authenticated only users
+- [[PR #11047]](https://github.com/kubevirt/kubevirt/pull/11047) [jschintag] Fix potential crash when trying to list USB devices on host without any
```
Comment (acardace) [inline docs/release_notes.md:83]: bugfix?

File: docs/release_notes.md [hunk 6]
```diff
@@ -5,6 +5,102 @@ hide:
...
+- [[PR #11001]](https://github.com/kubevirt/kubevirt/pull/11001) [fossedihelm] Allow `kubevirt.io:default` clusterRole to get,list kubevirts
+- [[PR #10918]](https://github.com/kubevirt/kubevirt/pull/10918) [orelmisan] VMClone: Emit an event in case restore creation fails
+- [[PR #10916]](https://github.com/kubevirt/kubevirt/pull/10916) [orelmisan] Fix the value of VMI `Status.GuestOSInfo.Version`
```
Comment (acardace) [inline docs/release_notes.md:86]: bugfix?

File: docs/release_notes.md [hunk 7]
```diff
@@ -5,6 +5,102 @@ hide:
...
+- [[PR #10596]](https://github.com/kubevirt/kubevirt/pull/10596) [mhenriks] Disable HTTP/2 to mitigate CVE-2023-44487
+- [[PR #10582]](https://github.com/kubevirt/kubevirt/pull/10582) [orelmisan] Remove leftover NonRootExperimental feature gate
+- [[PR #10567]](https://github.com/kubevirt/kubevirt/pull/10567) [awels] Attachment pod creation is now rate limited
+- [[PR #10526]](https://github.com/kubevirt/kubevirt/pull/10526) [cfilleke]  Documents steps to build the KubeVirt builder container
+- [[PR #10479]](https://github.com/kubevirt/kubevirt/pull/10479) [dharmit] Ability to run scripts through hook sidecardevice
+- [[PR #10244]](https://github.com/kubevirt/kubevirt/pull/10244) [hshitomi] Added “adm” subcommand under “virtctl”, and “log-verbosity" subcommand under “adm”. The log-verbosity command is: to show the log verbosity of one or more components, to set the log verbosity of one or more components, and to reset the log verbosity of all components (reset to the default verbosity (2)).
+- [[PR #10046]](https://github.com/kubevirt/kubevirt/pull/10046) [victortoso] Add v1alpha3 for hooks and fix migration when using sidecars
```
Comment (acardace) [inline docs/release_notes.md:102]: bugfix?

File: docs/release_notes.md [hunk 8]
```diff
@@ -5,6 +5,102 @@ hide:
...
+- [[PR #10866]](https://github.com/kubevirt/kubevirt/pull/10866) [AlonaKaplan] Raise an error in case passt feature gate or API are used.
+- [[PR #10800]](https://github.com/kubevirt/kubevirt/pull/10800) [AlonaKaplan] Support macvtap as a binding plugin
+- [[PR #10425]](https://github.com/kubevirt/kubevirt/pull/10425) [ormergi] Introduce network binding plugin for Passt networking, interfacing with Kubevirt new network binding plugin API.
+
+### SIG-infra
+- [[PR #11025]](https://github.com/kubevirt/kubevirt/pull/11025) [0xFelix] Allow unprivileged users read-only access to VirtualMachineCluster{Instancetypes,Preferences} by default.
+- [[PR #10922]](https://github.com/kubevirt/kubevirt/pull/10922) [kubevirt-bot] Updated common-instancetypes bundles to v0.4.0
+
+### SIG-scale
+- [[PR #10571]](https://github.com/kubevirt/kubevirt/pull/10571) [tiraboschi] vmi memory footprint increase by 35M when guest serial console logging is turned on (default on).
```
Comment (acardace) [inline docs/release_notes.md:66]: I think this is just sig-compute

File: docs/release_notes.md [hunk 9]
```diff
@@ -5,6 +5,102 @@ hide:
...
+- [[PR #10570]](https://github.com/kubevirt/kubevirt/pull/10570) [machadovilaca] Fix LowKVMNodesCount not firing
+- [[PR #10418]](https://github.com/kubevirt/kubevirt/pull/10418) [machadovilaca] Add total VMs created metric
+
+### Uncategorized
+- [[PR #11144]](https://github.com/kubevirt/kubevirt/pull/11144) [0xFelix] virtctl: Specifying size when creating a VM and using --volume-import to clone a PVC or a VolumeSnapshot is optional now
+- [[PR #11122]](https://github.com/kubevirt/kubevirt/pull/11122) [brianmcarey] Update runc dependency to v1.1.12
+- [[PR #11068]](https://github.com/kubevirt/kubevirt/pull/11068) [brianmcarey] Update container base image to use current stable debian 12 base
+- [[PR #11050]](https://github.com/kubevirt/kubevirt/pull/11050) [fossedihelm] restrict default cluster role to authenticated only users
+- [[PR #11047]](https://github.com/kubevirt/kubevirt/pull/11047) [jschintag] Fix potential crash when trying to list USB devices on host without any
+- [[PR #11001]](https://github.com/kubevirt/kubevirt/pull/11001) [fossedihelm] Allow `kubevirt.io:default` clusterRole to get,list kubevirts
```
Comment (acardace) [inline docs/release_notes.md:84]: sig-compute

File: docs/release_notes.md [hunk 10]
```diff
@@ -5,6 +5,102 @@ hide:
...
+- [[PR #10918]](https://github.com/kubevirt/kubevirt/pull/10918) [orelmisan] VMClone: Emit an event in case restore creation fails
```
Comment (acardace) [inline docs/release_notes.md:85]: sig-compute

File: docs/release_notes.md [hunk 11]
```diff
@@ -5,6 +5,102 @@ hide:
...
+- [[PR #10916]](https://github.com/kubevirt/kubevirt/pull/10916) [orelmisan] Fix the value of VMI `Status.GuestOSInfo.Version`
+- [[PR #10914]](https://github.com/kubevirt/kubevirt/pull/10914) [brianmcarey] KubeVirt is now built with go 1.21.5
+- [[PR #10898]](https://github.com/kubevirt/kubevirt/pull/10898) [matthewei] vmi status's guestOsInfo adds `Machine`
```
Comment (acardace) [inline docs/release_notes.md:88]: sig-compute

**Review by acardace — COMMENTED** [review]
Hey @aburdenthehand, left a few comments, please double check if they make sense.

File: docs/release_notes.md (hunk 7 above)
Comment (aburdenthehand) [reply docs/release_notes.md:102]: Alice previously suggested moving this out of bugfix as it also adds alpha1

File: docs/release_notes.md (hunk 6 above)
Comment (aburdenthehand) [reply docs/release_notes.md:86]: I've moved it as suggested but: Is it a bugfix if there's no bug? What is the sound of one hand clapping? :)

**Review by dhiller — COMMENTED** [review]
/approve

/hold to be removed when ready

**Comment by aburdenthehand** [comment]
I think it's good to print. Removing hold.

**Review by phoracek — APPROVED** [review]
/lgtm

**Pull request merged**